.github/workflows/ci.yml

```
@@ -91,6 +91,8 @@ jobs:
   build-system-dynamic:
     strategy:
       fail-fast: false
+      matrix:
+        libgit2: [ '1.0.0', '1.0.1', '1.1.0' ]
```

```suggestion
        libgit2: [ '1.0.0', '1.1.0' ]
```

we only need to consider the endpoints of the supported revisions to avoid overstraining the CI.

---

Sure